packages/core/src/controller.ts

```diff
@@ -38,7 +38,6 @@ export const harnessController = (Controller: ControllerInstance) => async (
   res: NextApiResponse,
 ) => {
   const startTime = new Date().getTime()
-  console.log('')
```

Comment:
Leave this one in — it adds a new line between logs so they are easier to read

Reply:
My bad, I thought it was useless. Why not simply use a newline character?

Reply:
Ha, yes you are right — I just hadn't thought about that :)